src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,31 @@
+package shop;
+
+import org.junit.jupiter.api.DisplayName;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.testng.Assert.*;
```

Comment:
Remove testng annotations and use junit only

Reply:
done